README.md

```diff
@@ -1 +1,59 @@
 # Terraform Provider for Looker
+
+This terraform provider interacts with the Looker API to configure Looker resources.
+
+## Documentation 
+
+This provider is not yet published to the terraform registry. Documentation for each resource and datasource supported can be found below.
+
+- [Resources](https://github.com/resolutionlife/terraform-provider-looker/tree/main/docs/resources) 
+- [Data sources](https://github.com/resolutionlife/terraform-provider-looker/tree/main/docs/data-sources)
+## Installation
+
+Terraform uses the Terraform Registry to download and install providers. This provider is not currently published on the terraform registry, so the provider binary must be built locally. 
+
+To build a binary locally, run `make install`. Then, run `terraform init` using the below configuration.
+
+```terraform
+terraform {
+  required_providers {
+    looker = {
+      source  = "terraform.example.com/local/looker"
+      version = "0.0.1"
+    }
+  }
+}
+
+provider "looker" {
+    base_url      = "https://my-instance.cloud.looker.com"
+    client_id     = "my-client-id"
+    client_secret = "my-client-secret"
+}
+```
+
+```sh
+$ make install
+```
+```sh
+$ terraform init
+```
+
+## Environment Variables
```

Comment:
Would it be a good idea to add details about the `TF_LOG_PROVIDER` env var here as well?